pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/impl/OverloadShedderTest.java

```diff
@@ -69,7 +68,7 @@ public void testBrokerNotOverloaded() {
         LocalBrokerData broker1 = new LocalBrokerData();
         broker1.setBandwidthIn(new ResourceUsage(500, 1000));
         broker1.setBandwidthOut(new ResourceUsage(500, 1000));
-        broker1.setBundles(Sets.newHashSet("bundle-1"));
+        broker1.getBundles().add("bundle-1");
```

Comment:
It looks like you need to clear this bundle, and then add the `bundle-1`.

Reply:
Nice catch. Updated. PTAL @nodece